### DISCUSSION TITLE
Naming Downloads

### DISCUSSION THREAD

**Discussion by Someguy2317** [aclap-dev/video-downloadhelper#1527] [Help and Q&A]
### Your question

Hi,

I used to use a very old version of the extension and coapp, and I recently upgraded my computer. 

I currently run 9.1.0.44 (stable) on the latest version of Firefox in Windows 11 home.

When I was downloading streams using the coapp, it used to open a dialog where I could change the name of the file before it started saving it. Making everything generic, it would originally have "watch TVSHOW season 1 on WEBSITE" and so I could change it to "TVSHOW S01E01" and so on since the coapp name would always be "watch TVSHOW season 1 on WEBSITE" from the start.

At the moment, I can't seem to find a way to make it have this behaviour.

I'm using the older UI since the new one doesn't appear to be as flexible/informative.

### Addon & CoApp version and details

version: 9.1.0.44
channel: stable
build date: 2024-09-04
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-CA
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:133.0) Gecko/20100101 Firefox/133.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\REDACTED",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by Someguy2317** [comment]
Just to add, I tried looking for an existing answer but either didn't use the correct terms or such an answer doesn't exist at the moment.

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

You'll want to read #665 & #814 carefully.  I think eventually you will change your opinion about the new user interface.  You will eventually realize it is indeed a great improvement over the old one.  I know change is difficult.  But you can eventually get over that mental block.

**Reply by Someguy2317** [reply]
Wow, I'm reading what you suggested and it's a goldmine of information, I've bookmarked both discussions, thanks again!

**Comment by Someguy2317** [comment]
Thank you very much, I'll read what you suggested and hopefully it'll clear everything up.

Appreciate the help!
